### ISSUE TITLE
Inverse Folding Task Evaluation

### ISSUE THREAD

**Issue by cucumber-potato** [JocelynSong/SurfPro#11]
Hi Dr. Song,
      Thank you for providing me with the data and model weights before, but I still couldn't get the ideal results after using these data and model weights.
       I conducted Inverse Folding Task Evaluation using the weights of my trained model and the model weights you provided, and the results are shown in the following figure. Thank you for your response！

This is bash generation_cath42.sh using the weights of the model I trained
![Image](https://github.com/user-attachments/assets/23758b8f-e0f7-4900-b6b1-3bd015073fdc)

![Image](https://github.com/user-attachments/assets/b7207107-ce25-4792-b90c-69d059ae7e0f)
This is bash generation_cath42.sh using the weights you gave to the model

![Image](https://github.com/user-attachments/assets/c7905fa0-a5d9-48a7-aefc-7645272c520d)

![Image](https://github.com/user-attachments/assets/92d05142-f65a-4c10-a113-4abe000b42cc)

**Comment by JocelynSong** [comment]
Hi, I just checked the code and found that I wrongly made all input features be zero in inference. To fix this bug, you can delete this line https://github.com/JocelynSong/SurfPro/blob/3d92b76114e5a0e0a09299a1da1541406a063330/fairseq/models/surface_protein_model.py#L552. Really sorry about that!

By the way, the pretrained model are not specifically trained on the CATH4.2 dataset, which is used for downstream task finetuning. If you want to test CATH4.2 performance, you can use the model weights trained on CATH 4.2 dataset, which is available at https://drive.google.com/file/d/1xl_3xUrzjZCYcl5C5eUKjJd5VoRG28NQ/view?usp=drive_link. I've just regenerated the CATH test set and the results should be good as below. Thanks!

<img width="1213" alt="Image" src="https://github.com/user-attachments/assets/14d94f41-5b35-4433-b498-5beb7be3d2d5" />

**Comment by cucumber-potato** [comment]
Hi Dr. Song,
After the code was modified, good results were obtained. Thank you for providing assistance.

![Image](https://github.com/user-attachments/assets/115bd322-18e0-408d-92e2-7c7b86964ae8)

**Comment by JocelynSong** [comment]
Nice! Happy to hear that!
